scripts/wrap_sundials.jl

```diff
@@ -136,11 +156,11 @@ function wrap_sundials_api(expr::Expr)
         expr.args[1].head == :call
         func_name = string(expr.args[1].args[1])
         convert_required = false
-        if occursin(r"^(ARK|CV|KIN|IDA|SUN|N_V)", func_name)
+        if occursin(r"^(ARK|ERK|MRI|CV|KIN|IDA|SUN|N_V)", func_name)
 	        @show func_name
-            if occursin(r"Create$", func_name)
+            if occursin(r"Create$", func_name) && !occursin(r"Butcher", func_name)
```

Comment:
does this mean that tableaus aren't wrapped?

Reply:
They weren't wrapped either in the previous script. I can look deeper into it but I can't tell why they weren't wrapped before.

Reply:
alright, no worries.

Reply:
I thought we were able to get tableaus before: https://docs.sciml.ai/latest/solvers/ode_solve/#ode_solve_sundials-1 `itable` and such. But this must be different, this must be about defining your own tableau.

Reply:
Just for reference, the current wrapper is based on this one 
https://github.com/SciML/Sundials.jl/blob/master/src/wrap_sundials.jl